pkg/scheduler/apis/config/types.go

```diff
@@ -164,6 +164,9 @@ type Plugins struct {
 	// NormalizeScore is a list of plugins that should be invoked after the scoring phase to normalize scores.
 	NormalizeScore *PluginSet
 
+	// Cleanup is a list of plugins that should be invoked after the scheduling framework is completed or aborted.
```

Comment:
We have a documentation convention, so it should be:
```suggestion
	// `cleanup` is a list of plugins that should be invoked after the scheduling framework is completed or aborted.
```
Please. also fix for other type properties.

Reply:
I don't think this is valid, please take a look at the other comments in this file.

Reply:
Yep, sorry, my bad, it's valid only for json/grpc tagged types, sorry.